ExampleApplication/TWTSampleViewController.m

```diff
@@ -52,6 +56,13 @@ - (void)viewDidLoad
     self.examples = [NSArray arrayWithContentsOfFile:toastSamplesURLString];
     
     [self.tableView registerClass:[UITableViewCell class] forCellReuseIdentifier:twt_cellIdentifier];
+
+    // TWTLocationController
+    // Update info.plist with either NSLocationWhenInUseUsageDescription and/or NSLocationAlwaysUsageDescription
+    [[TWTLocationController sharedInstance] fetchCurrentLocationWithCompletion:^(CLLocation *location, NSError *error) {
+        NSTimeInterval diff = [[NSDate date] timeIntervalSinceDate:location.timestamp];
```

Comment:
Not crazy about the name “diff.” What about something like, timeIntervalSinceUpdate or elapsedTime or whatever this is trying to tell us.